apps/storefront/tests/utils/eventManager.test.ts

```diff
@@ -1,57 +1,69 @@
-import { B2BEvent } from '@b3/hooks';
-
 import CallbackManager from '@/utils/b3CallbackManager';
 
 describe('Callback manager', () => {
-  const callbackManager = new CallbackManager();
-
-  beforeEach(async () => {
-    callbackManager.callbacks.clear();
-  });
-
   test('addEventListener register events if key does not exist', () => {
```

Comment:
the wording on these tests is all a bit weird

are we just trying to describe that the _first and only_ callback is ran? 
e.g. "key does not exist" == "nothing else has subscribed yet" ..?

seeing as this works the same when there are or are not subscribers, having this test at all seems misleading, like there should be special behaviour

Reply:
test name is not the best, no. It should probably read something like `registers a callback`

the test was probably written because there's some special case when implementing the callback manager:

```tsx
    if (!this.callbacks.has(callbackKey)) {
      this.callbacks.set(callbackKey, [callback]);
    } else {
```